src/crimson/osd/pg.h

```diff
@@ -617,6 +617,12 @@ class PG : public boost::intrusive_ref_counter<
   OSDriver osdriver;
   SnapMapper snap_mapper;
 
+  ghobject_t make_snapmapper_oid() {
+    return ghobject_t(hobject_t(
+      sobject_t(
+       object_t("snapmapper"),
+       0)));
+  }
```

Comment:
Let's move this to `crimson/osd/osd_meta.cc` where we already create the osdmap/superblock/final_pool_info oids.
This will require introducing `void OSDMeta::create_snapmapper`.

Reply:
Since we are no longer using OSD-wide snapmapper object, this may not be necessary, am I right?